compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -2168,6 +2168,25 @@ private boolean isModuleLevelVar(BSymbol symbol) {
         return symbol.getKind() == SymbolKind.VARIABLE && symbol.owner.getKind() == SymbolKind.PACKAGE;
     }
 
+    public Set<BVarSymbol> getConfigVarSymbols(BPackageSymbol packageSymbol) {
```

Comment:
Should the method name reflect we're looking up symbols from imported modules also?